### ISSUE TITLE
Styleguide

### ISSUE THREAD

**Issue by monarchwadia** [MintbeanHackathons/MintbeanPlatform#16]
We need a simple brand styleguide page on the mintbean.io
- Primary colors
- Fonts
- Logos

Keep it simple for now

**Comment by srsexton94** [comment]
I can take this on if you like!

**Comment by shohei-shibata** [comment]
I'm no design person but happy to help on this stuff. I've done some basic style guide and wireframes on [Figma](https://www.figma.com/)... possible tool suggestion :)

**Comment by oscarnava** [comment]
I could also help by creating an SVG version of the logo, and several resolution PNG's.

**Comment by oscarnava** [comment]
I've already created the SVG version and 300, 150, 96 & 72 dpi PNG's. I won't submit them yet until these issues get assigned, but I just wanted to let you know I have them ready if/when needed. 😁

**Comment by srsexton94** [comment]
@shohei-shibata yes! Figma is a great tool - I've stuck with Balsamiq as low-res style always served my purposes well - but I think for a big collab the higher res would be useful!
(plus apparently I was on a free trial for Balsamiq and just lost access lol)

And @oscarnava that sounds great! Thanks so much for doing that!

I could fork/branch and create a markdown file to compile this stuff as we build it if that's helpful

**Comment by srsexton94** [comment]
For future use (checklist?) I posit we include:

* A general branding statement (speaks to mission/values/vision - helps align the design to visually represent the org)
* Typography (`h1`--`h5`, `p`, and `a` elements; include font-weight, family, and color)
* Color Palate (primary, secondary, [tertiary], and neutrals)
* Spacing (how much "whitespace"? any layout grids we use? maybe point to BootstrapVue?)
* Voice considerations (is text content meant to be funny? formal? welcoming? professional? etc.)
* Icons (including logo - TY @oscarnava! - as well as any visual indicator icons, external link markers, etc.. any specific icon library?)
* Imagery (give stylistic description as well as examples - update with image files of images used as the project develops)
* Buttons (here would be good to point to BootstrapVue!)
* If we have any Forms it'd be good to also specify hierarchy, feedback, validation, etc..

**Comment by shohei-shibata** [comment]
@srsexton94 I have just worked on another markdown-based page ("For Educators") so it should be pretty quick for me to add a similar one for this, with just a placeholder content for now. 

As for assignment - Looks like issues aren't being assigned to anyone. I wasn't "assigned" on the other issue I just finished :) 

Maybe eventually that'll happen but probably not until the whole project workflow settles down more. I'd say let's just get started?

**Comment by srsexton94** [comment]
Oh goodness I'm sorry I missed this - yes absolutely lets. 
@shohei-shibata have you already done so?  I'm not seeing anything in the repo but I imagine it might not have been merged in?  What kind of workflow do you think would work well?

**Comment by shohei-shibata** [comment]
@srsexton94 I have a placeholder page done, but didn't submit a PR yet. I'll go ahead and do that now. 

As for where to go from there... did you want to take on creating the style guide content like typography, colors, etc?

**Comment by srsexton94** [comment]
Yes absolutely! I'll tackle that tomorrow morning - if the PR isn't accepted by then is there a way I could best edit `style-guide/index.md `?

**Comment by shohei-shibata** [comment]
Awesome! Yeah the PR isn't merged yet... I'm thinking you could either fork my fork or, edit the markdown file somewhere else and paste into the code after the PR is merged?

**Comment by srsexton94** [comment]
Hey so I ended up just copy/pasting to my own fork of the main repo and pushed, but left a note in the PR that it should be merged after #30

**Comment by shohei-shibata** [comment]
@srsexton94  sounds good to me!

**Comment by monarchwadia** [comment]
Hi guys, I've left comments on the PR, please take a look :-)
